plenum/server/node.py

```diff
@@ -2777,11 +2776,16 @@ def verifySignature(self, msg):
             typ = ''
             req = msg
 
+        key = None
```

Comment:
Need to have a key for request inside of propagate here to use auth cache of ReqAuthenticator.